packages/ckeditor5-image/src/imageinsert/imageinsertui.ts

```diff
@@ -36,6 +42,8 @@ import ImageUtils from '../imageutils.js';
  *
  * Adds the `'insertImage'` dropdown to the {@link module:ui/componentfactory~ComponentFactory UI component factory}
  * and also the `imageInsert` dropdown as an alias for backward compatibility.
+ *
+ * Adds the `'menuBar:insertImage'` sub-menu to the {@link module:ui/componentfactory~ComponentFactory UI component factory}.
```

Comment:
We also added new components in ckbox and ckfinder and AFAICS their API docs are not updated as well.

I also wanted you to go through existing guides but since core team is now reworking them AND we need to do other tasks related to docs (update demos, update all other guides not only images), then let's skip it for now.